app/models/User.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
```

Comment:
change users to User since that parameter corresponds to the name of the model, which in this case is User

Reply:
And as the name of the model will be different from the name of the table, you should find a way to relate them